### DISCUSSION TITLE
Report:  "Fake" RPM Package Running!

### DISCUSSION THREAD

**Discussion by horus-pclos** [lgblgblgb/xclcd#310] [General]
Greetings from the U.S. Gulf Coast.

I seem to have the Xemu RPM package running under PCLinuxOS.  Particulars:

I haven't yet pulled a ROM, and won't have time for that until later, but thought you'd want a report of this partial success.
uname -r reports kernel 5.14.15 is running.
System is PCLinuxOS 2021, updated last night.  Desktop is XFCE 4.16
Hardware is an old Dell Precision M4600, i5-2520M, with 4GB of RAM and Nvidia Quadro 2000M video.

Installation was dead simple:  download the package to ~/Downloads/Emulators then, as root, rpm -ivh xemu-current-1.x86_64.rpm.  No dependency issues noted during installation.
Running from terminal as normal user works.

Wanted to say THANK YOU for creating this emulator. It appears to be quite versatile, and I hope it will be useful in evaluating the MegaC65 environment.

**Comment by lgblgblgb** [comment]
Thank you for reporting back. Honestly, the problem with Xemu distribution that surely I cannot have enough time, resources, etc, to distribute Xemu for every possible Linux distributions (surely with the source, some can compile it at their own ...). The "fake RPM" package is called "fake" as it's a direct conversion (with the tool named "alien") from DEB package. So though the result is RPM, it may not meet with the dependency/version needs of any RPM based Linux distribution or other problems. But I'm glad if you say you had success with it.
